src/Configuration/Configuration.slnf

```diff
@@ -0,0 +1,35 @@
+{

```

Comment:
👀 nice how you slipped the .sln -> .slnf change in here 😈

Reply:
Yeah, I didn't want to update the sln ;P. It's somewhat moot since this code is mostly already moved/moving to dotnet/runtime.